packages/cfl/tests/test_finding.py

```
@@ -0,0 +1,917 @@
+        visibility="presented",
+    )
+
+    row = db_conn.execute(
+        "SELECT g.gate_type FROM findings f JOIN gates g ON f.gate_id = g.id"
+        " WHERE f.run_id=?",
+        (run_id,),
+    ).fetchone()
+    assert row["gate_type"] == "ship-challenge"
+
+
+def test_record_finding_run_id_none_writes_successfully(db_conn, capsys):
```

Ruff (0.16.1)

[warning] Unused function argument: `capsys`

(ARG001)